src/Chainweb/Pact/PactService.hs

```diff
@@ -426,15 +426,14 @@ readAccountGuard pdb account
 --
 readRewards
     :: HasChainGraph v
-    => v -> IO (HashMap BlockHeight P.ParsedDecimal)
+    => v -> (HashMap BlockHeight P.ParsedDecimal)
 readRewards v = do
-    rs <- LBS.readFile "rewards/miner_rewards.csv"
-    case CSV.decode CSV.NoHeader rs of
-      Left e -> internalError
+    case CSV.decode CSV.NoHeader (toS rawMinerRewards) of
+      -- TODO Not sure what to do here
+      Left e -> error
```

Comment:
`error` is perfectly fine. We want this to be a 'fail fast and hard' scenario. We can also now `{-# INLINE readRewards #-}`